web/src/controller/organization_controller.rs

```diff
@@ -22,7 +22,9 @@ impl OrganizationController {
     /// --request GET \
     /// http://localhost:4000/organizations
     pub async fn index(State(app_state): State<AppState>) -> impl IntoResponse {
-        let organizations = OrganizationApi::find_all(&app_state).await;
+        let organizations = OrganizationApi::find_all(&app_state)
```

Comment:
I think I prefer keeping the `unwrap_or_default`  detail inside of the entity_api context since to me it's an implementation detail and will always be the behavior we're expecting with a `find_all`. Open to your thoughts on this.

Reply:
That's ok, this was only recognizing that `unwrap_or(vec![])` vs `unwrap_or_default()` should return the same thing. But if you think moving any unwrap into the EntityAPI for Organization is even stronger, I'm good with that. Let's take a look at this further together.